src/Pages/License/Generate/CustomerDetails.jsx

```diff
@@ -1,12 +1,60 @@
+import React from "react";
 import Grid from "@mui/material/Grid";
 import Typography from "@mui/material/Typography";
 import TextField from "@mui/material/TextField";
+import Autocomplete, { createFilterOptions } from "@mui/material/Autocomplete";
 import { useContext } from "react";
 import { CustomCard, CustomCardContent } from "./GenerateLicense";
 import { AppContext } from "App";
 
+const customerNameList = [
```

Comment:
Add TODO comment that hard coded data should be removed once APIs are available

Reply:
done